Tests/Unit/EmogrifierTest.php

```diff
@@ -692,6 +692,9 @@ public function cssDeclarationWhitespaceDroppingDataProvider()
             'space before trailing semicolon' => [' color:#000 ;', 'color: #000;'],
             'space after trailing semicolon' => [' color:#000; ', 'color: #000;'],
             'space after property value, no trailing semicolon' => [' color:#000; ', 'color: #000;'],
+            'newline before property value, trailing semicolon' => ["\ncolor:#222; ", 'color: #222;'],
+            'newline before colon and after value, trailing semicolon' => ["color\n:#333\n; ", 'color: #333;'],
```

Comment:
Please let's split this up in two separate lines:
1. 'newline before colon, trailing semicolon'
2. 'newline after value, trailing semicolon'

We also should have a test for the case of a newline after the semicolon.